t/BikeSharing/IndegoPHL.t

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+ddg_spice_test(
+    [qw( DDG::Spice::BikeSharing::IndegoPHL )],
+    'philly indego' => test_spice(
+        '/js/spice/bike_sharing/indego_phl/philly',
+        call_type => 'include',
+        caller => 'DDG::Spice::BikeSharing::IndegoPHL',
+    is_cached => 1,
```

Comment:
@marianosimone it's totally not your fault for misunderstanding. The caching params really need better names ;) -- There's a task to make them more descriptive, we just haven't gotten around to it. Right now it's really hard to reason what they're for unless you dig into the docs...